### ISSUE TITLE
Staging Review finding: Unnecessary `aria-label` that may create a barrier for voice users

### ISSUE THREAD

**Issue by shiragoodman** [department-of-veterans-affairs/va.gov-team#98480]
_Need help? Please review [how to read a Staging Review ticket](https://depo-platform-documentation.scrollhelp.site/collaboration-cycle/anatomy-of-a-staging-review-issue-ticket). Tag `@platform-governance-team-members` on Slack if you need further assistance._

## Product Information
**Team:** IIR
**Product:** Veteran New Account Flow within Authenticated Experience
**Feature:** Welcome Pathway V1

## Findings details
**VA.gov Experience Standard - issue:** [User can't complete a necessary flow or hits a dead end.](https://design.va.gov/about/experience-standards/#user-can-complete-necessary-flows)
**VA.gov Experience Standard - category:** [Usability](https://design.va.gov/about/experience-standards/#category-usability)
**Launch-blocking:** No
**Design System review:** No
**Collab Cycle Reviewer:** @briandeconinck (Accessibility)

### Description

After successfully adding your contact information and reaching [/review-information/confirmation](https://staging.va.gov/my-va/welcome-va-setup/review-information/confirmation), there's a success alert with a primary action link with text "Go to your notification settings in your VA.gov profile."

The `va-link-action` component also has a `label="Notification settings link"`, which maps to an `aria-label` of "Notification settings link."

An `aria-label` replaces the link text in the accessibility tree, meaning screen readers will announce "Notification settings link" rather than "Go to your notification settings in your VA.gov profile." Most screen readers will also announce the type of element --- meaning when I tested, it was announced as "Notification settings link link," which is clearly a degraded experience from the full descriptive link text.

But in addition to the screen reader experience, there's an impact for voice command / speech recognition software users. When a sighted voice user sees the visible link text and speaks the command "Click go to your notification settings in your VA.gov profile," some voice software will look for matches in the accessibility tree --- which means they'll look for a match for "go to your notification settings in your VA.gov profile" but the link with that name has been renamed "Notification settings link." Some software (eg. Dragon on Windows) might be smart enough to get a match just from "notification settings," but other software (eg. Apple Voice Control) likely won't find a match.

<details><summary>Link, screenshot or steps to recreate</summary>

![Success alert with heading "Contact information added to your profile" and a primary action link reading "Go to your notification settings in your VA.gov profile."](https://github.com/department-of-veterans-affairs/va.gov-team/blob/master/platform/working-with-vsp/vsp-collaboration-cycle/staging-review-images/82257-Brian-3.png?raw=true)

Currently coded as:
```
<va-link-action href="https://www.va.gov/profile/notifications" label="Notification settings link" text="Go to your notification settings in your VA.gov profile" class="hydrated"></va-link-action>
```

</details>

## Recommended action

I don't think the `aria-label` adds any value here. Remove the `label` prop from the link.

## References

- **Accessibility Defect Severity:** [2: High. Should be fixed in 1-2 sprints post-launch.](https://depo-platform-documentation.scrollhelp.site/developer-docs/accessibility-defect-severity-rubric#Accessibilitydefectseverityrubric-a11y-defect-2)
- **WCAG Success Criteria:** N/A
- **Modality:** Voice
- **Design System Component:** N/A
- **Design System Pattern or Template:** N/A
- **Design System Foundation:** N/A
- **Content Style Guide:** N/A
- **Context:** N/A

---

### Next Steps for the VFS Team

- [ ] **Questions?** For the most timely response, comment on Slack in your team channel tagging `@platform-governance-team-members` with any questions or to get help validating the issue.
- [ ] **Ticket ownership:** If you believe that this issue is out of scope, not your team's responsibility, or a Design System issue, comment and tag `@platform-governance-team-members` on your team channel in Slack to provide an explanation and who you believe is responsible. The Governance team will follow up.
- [ ] **Close this ticket** when the issue has been resolved or validated by your Product Owner.

**Comment by mcommons10** [comment]
This has been addressed. Closing ticket.
